schema.production.kf

```diff
@@ -20,191 +22,165 @@ use idos {
     chain: 'etherlink'
 } as idos_etherlink;
 
-table humans {
-    id text primary minlen(36) maxlen(36) notnull unique //lengths to help enforce uuidv4
-}
-
-table human_attributes {
-    id text primary minlen(36) maxlen(36) notnull unique,
-    human_id text minlen(36) maxlen(36) notnull,
-    attribute_key text notnull,
-    value text notnull,
-    #human_attributes_human_id index(human_id),
-    foreign_key (human_id) references humans(id) on_delete cascade
-}
+// TABLES
 
-table shared_human_attributes {
-    original_id text minlen(36) maxlen(36) notnull,
-    duplicate_id text minlen(36) maxlen(36) notnull,
-    #primary_key primary(original_id, duplicate_id),
-    foreign_key (original_id) references human_attributes(id) on_delete cascade,
-    foreign_key (duplicate_id) references human_attributes(id) on_delete cascade
+table humans {
+    id uuid primary,
+    issuer text notnull
 }
 
 table wallets {
-    id text primary minlen(36) maxlen(36) notnull unique,
-    human_id text minlen(36) maxlen(36) notnull,
+    id uuid primary,
+    human_id uuid notnull,
     address text notnull,
     public_key text,
     wallet_type text notnull,
     message text,
     signature text,
+    issuer text,
     #wallets_human_id index(human_id),
-    #wallets_evm_scan index(wallet_type, address),
-    #wallets_near_scan index(wallet_type, public_key),
+    #wallets_evm_scan unique(wallet_type, address),
```

Comment:
This will mess up adding NEAR wallets, since we add an entry for each FullAccess public key with the same address.

Ideally, this index should be a partial one on `wallet_type = 'EVM'`.

Reply:
Let's just use `DISTINCT` until partial indexes are supported. Regarding making a mess with the human_id <-> wallets, we're not worse off.

Júlio also suggested that we could perhaps separate the EVM and NEAR wallets into two separate tables.